docs/processors.md

```diff
@@ -121,3 +121,5 @@ final class BoardGameResource implements ResourceInterface
 ```
 
 **[Go back to the documentation's index](index.md)**
+
+**[> Next chapter: Responders](responders.md)**
```

Comment:
Where are responders.md?

Reply:
https://github.com/Sylius/SyliusResourceBundle/blob/1e3e360f7b0fffb31dc8ff679159f45ceeecc88a/docs/responders.md

Reply:
You could have a look on the preview.